src/main/java/org/commonjava/service/metadata/handler/PathUtils.java

```diff
@@ -0,0 +1,83 @@
+package org.commonjava.service.metadata.handler;
+
+import org.commonjava.event.common.EventMetadata;
+
+public final class PathUtils {
+    public static final String ROOT = "/";
+    private static final String[] ROOT_ARRY = new String[]{"/"};
+
+    private PathUtils() {
+    }
+
+    public static String[] parentPath(String path) {
+        String[] parts = path.split("/");
+        if (parts.length < 2) {
+            return ROOT_ARRY;
+        } else {
+            String[] parentParts = new String[parts.length - 1];
+            System.arraycopy(parts, 0, parentParts, 0, parentParts.length);
+            return parentParts;
+        }
+    }
+
+    public static String normalize(String... path) {
```

Comment:
I think this is copied from somewhere, I don't remember. but we'd better remove it from our code.

Reply:
See, I will update it.